application/src/test/java/org/thingsboard/server/controller/AbstractWebTest.java

```diff
@@ -699,4 +711,49 @@ protected <T extends HasId<? extends UUIDBased>> ListenableFuture<List<ResultAct
         return Futures.allAsList(futures);
     }
 
+    protected void tesEntityDaoWithRelationsOk(EntityId entityIdFrom, EntityId entityTo, String urlDelete) throws Exception {
```

Comment:
1. Method name is confusing about Dao and has a typo.
2. let's test itself and stay in the respective class. As delete entity implemented in different classes and may and should differ, the test logic has to be decoupled in respective classes too.
3. It is enough to extract small common methods like `createEntityRelation` and 'findRelationsTo', etc.

Reply:
Rename - ok